how-to-use-azureml/deployment/accelerated-models/accelerated-models-training.ipynb

```diff
@@ -0,0 +1,849 @@
+    "                                        in_labels: label_test,\n",
+    "                                        K.learning_phase(): 0})\n",
+    "    return accuracy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"execution\"></a>\n",
+    "## 6. Execute steps\n",
+    "You can run through the Transfer Learning section, then skip to Model Deployment. By default, because the custom weights section takes much longer for training twice, it is all saved as Raw NBConvert mode. This means it is not executable, but you can change them to Code type to run through them.\n",
+    "\n",
+    "<a id=\"transfer-learning\"></a>\n",
+    "### 6.a. Training using Transfer Learning"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Launch the training\n",
+    "tf.reset_default_graph()\n",
+    "sess = tf.Session(graph=tf.get_default_graph())\n",
+    "\n",
+    "with sess.as_default():\n",
+    "    in_images, image_tensors, features, preds, featurizer = construct_model(quantized=True)\n",
+    "    train_model(preds, in_images, img_train, label_train, is_retrain=False, train_epoch=10, learning_rate=0.01)    \n",
+    "    accuracy = test_model(preds, in_images, img_test, label_test)  \n",
+    "    print(\"Accuracy:\", accuracy)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Save Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_name = 'resnet50-catsanddogs-tl'\n",
+    "model_save_path = os.path.join(saved_model_dir, model_name)\n",
+    "\n",
+    "tf.saved_model.simple_save(sess, model_save_path,\n",
+    "                               inputs={'images': in_images},\n",
+    "                               outputs={'output_alias': preds})\n",
+    "\n",
+    "input_tensors = in_images.name\n",
+    "output_tensors = preds.name\n",
+    "\n",
+    "print(input_tensors)\n",
+    "print(output_tensors)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"custom-weights\"></a>\n",
+    "### 6.b. Traning using Custom Weights\n",
+    "\n",
+    "Because the quantized graph defintion and the float32 graph defintion share the same node names in the graph definitions, we can initally train the weights in float32, and then reload them with the quantized operations (which take longer) to fine-tune the model.\n",
+    "\n",
+    "First we train the model with custom weights but without quantization. Training is done with native float precision (32-bit floats). We load the training data set and batch the training with 10 epochs. When the performance reaches desired level or starts decredation, we stop the training iteration and save the weights as tensorflow checkpoint files. "
+   ]
+  },
+  {
+   "cell_type": "raw",
```

Comment:
Please check the markdown rendering of these cells, should it be code instead?

Reply:
is updated